components/NewTripForm/index.js

```diff
@@ -1,56 +1,98 @@
+import { Fragment, useState } from "react";
 import styled from "styled-components";
 
 export default function NewTripForm() {
   // default value till flexible trip duration will be implemented
   const tripDurationInDays = 5;
 
-  // The inputs fields "Day title" and "Activities" need to be displayed based on the duration of the trip in days. So I use a loop here and call the function in line 45
+  // state holding successfull save messages for form
+  const [saveMessages, setSaveMessages] = useState([]);
+
+  // The inputs fields "Day title" and "Activities" need to be displayed based on the duration of the trip in days. So I use a loop for that and call the function which returns the right amount of input fields in line 45
   function createMultipleDays() {
     const tripDays = [];
 
     for (let i = 0; i < tripDurationInDays; i++) {
       tripDays.push(
-        <div>
-          <label htmlFor="title">
-            Day title:
-            <input type="text" name="title" id="title" />
-          </label>
-          <label htmlFor="activities">
-            Activities:
-            <textarea name="activities" id="activities"></textarea>
-          </label>
-        </div>
+        <Fragment key={`day-${i}`}>
```

Comment:
I don't need the Fragment here as it was only for holding a key attribute.
The key attribute can be added to the